docs/validators/staking-v4.md

```diff
@@ -0,0 +1,297 @@
+    - a global number of 1320 waiting validators to join the active list, split in 320 nodes/shard
+- All **shuffled out** nodes from eligible list will be sent to auction list in order to take part in the auction
+  selection. The more topUp an owner has, the higher the chances of having his auction nodes selected will be.
+- Based on the _soft auction selection_ (see next section), all **qualified** nodes from **auction** will be distributed
+  to waiting list (depending on the `available slots`). The other **unqualified** nodes will remain in auction and wait
+  to be selected in the next epoch, if possible. The number of available slots is based on the number of shuffled out
+  nodes and other nodes leaving the network (e.g.: `unstake/jail`). It guarantees that the waiting list is filled and
+  the nodes configuration is maintained.
+- Distribution from `waiting` to `eligible` list will remain unchanged
+
+![](/validators/stakingV4/stakingV4-phase3.png)
+
+## Staking v4. Soft auction selection mechanism
+
+Nodes from the auction list will be selected to be distributed in the waiting list based on the **soft auction**
+mechanism.
+For each owner, based on his topUp, we compute how many validators he would be able to run by distributing his total
+topUp per fewer nodes(considering we would not select all of his auction nodes, but only a part of them). This mechanism
+ensures that for each owner we select as many nodes as possible, based on the **minimum required topUp** to fill the
+`available slots`. This is a global selection, not per shard. We preselect the best global nodes at the end of epoch.
+
+Suppose we have the following auction list, and 3 available slots:
+
+![](/validators/stakingV4/soft-auction1.png)
+
+```
++--------+------------------+------------------+-------------------+--------------+-----------------+-------------------------+
+| Owner  | Num staked nodes | Num active nodes | Num auction nodes | Total top up | Top up per node | Auction list nodes      |
++--------+------------------+------------------+-------------------+--------------+-----------------+-------------------------+
+| owner1 | 3                | 2                | 1                 | 3666         | 1222            | pubKey1                 |
+| owner2 | 3                | 1                | 2                 | 2555         | 851             | pubKey2, pubKey3        |
+| owner3 | 2                | 1                | 1                 | 2444         | 1222            | pubKey4                 |
+| owner4 | 4                | 1                | 3                 | 2666         | 666             | pubKey5, pubKe6, pubKe7 |
++--------+------------------+------------------+-------------------+--------------+-----------------+-------------------------+  
+```
+
+For the configuration above:
+
+- Minimum possible `topUp per node` = 666, considering `owner4` will have all of his **3 auction nodes selected**
```

Comment:
Suggestion:
I would change numbers to get rid of the 666

Reply:
Fixed